src/app/pages/IndexPage/index.test.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import IndexPage from '.';
+
+describe('IndexPage', () => {
+  shouldMatchSnapshot(
```

Comment:
Worth having another `describe` to group the snapshots as we will be adding assertions later on.

```suggestion
describe('Snapshots', () => {
  shouldMatchSnapshot(
```